angular/base/tslint.json

```diff
@@ -140,9 +140,12 @@
     "template-banana-in-box": true,
     "template-no-negated-async": true,
     "use-lifecycle-interface": true,
-    "use-pipe-transform-interface": true
+    "use-pipe-transform-interface": true,
+    
+    // Custom for Ionic Documentation
```

Comment:
Could you just remove the comments in the json?

Reply:
Yes, I did!